.github/workflows/no-response.yml

```diff
@@ -15,6 +15,7 @@ jobs:
     permissions:
       contents: read
       pull-requests: write
+      issues: write
```

Comment:
```suggestion
      issues: write
```

Reply:
Indeed, we only use this label for issues.

> Did you try it on your repo to verify it fixes the problem?

I have GitHub Actions disabled on my fork, but let me try if I can test it

Reply:
Seems to work on my fork: https://github.com/cherniavskii/material-ui-x/issues/4